docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -344,37 +388,39 @@ def review_viewing(self, full_display_map=generator_constants.FULL_DISPLAY):
     return content
 
   def attachment_yaml(self):
-    # Lets use the list-style, not dictionary style fields statement
-    # To avoid duplicate key error
+    # We use the list-style, not dictionary style fields statement
+    # to avoid duplicate key error
     if hasattr(self, 'paired_yesno') and self.paired_yesno:
       content = ''
       for raw_name in self.raw_field_names:
         var_name = remove_multiple_appearance_indicator(varname(raw_name))
         if var_name.endswith('_yes'):
-          content += '      - "{}": ${{ {} }}\n'.format(raw_name, self.final_display_var)
+          content += '- "{}": ${{ {} }}'.format(raw_name, self.final_display_var)
         elif var_name.endswith('_no'):
-          content += '      - "{}": ${{ not {} }}\n'.format(raw_name, self.final_display_var)
+          content += '- "{}": ${{ not {} }}'.format(raw_name, self.final_display_var)
```

Comment:
This still needs a new line: otherwise all of these pairs are on the same line and there's a scanner error. 

```
 - "receive_spam_yes": ${ receive_spam }- "receive_spam_yes__1": ${ receive_spam }- "receive_spam_yes__3": ${ receive_spam }- "receive_spam_no": ${ not receive_spam }- "receive_spam_no__2": ${ not receive_spam }
```

Reply:
https://github.com/SuffolkLITLab/docassemble-assemblylinewizard/commit/4f78770627dc24b353bb2b4dff7ec22b60c5b487